contracts/consensus/Consensus.sol

```diff
@@ -128,7 +129,7 @@ contract Consensus is MasterCopyNonUpgradable, CoreStatusEnum, ConsensusI {
     modifier onlyCore()
     {
         require(
-            isCore(msg.sender),
+            isCoreActive(msg.sender),
```

Comment:
To me, isCoreActive() should return true only if core lifetime is set active (excluding here genesis).
I would also propose to change modifier name to `onlyActiveCore`.

Reply:
`isCoreRunning` is explicitly proposed for this reason; it must be true for `genesis` and `active`